addon/components/popover-on-element.js

```diff
@@ -1,122 +1,14 @@
-import Ember from 'ember';
-import TooltipAndPopoverComponent from 'ember-tooltips/components/tooltip-and-popover';
```

Comment:
Am I correct in believing a component no longer exists at this path? If developers have used this component for their own JS or styling then does this create a breaking change?